_sass/blocks/_landing.scss

```diff
@@ -0,0 +1,140 @@
+.landing-section_open,
+.landing-section {
+  padding-bottom: $base-padding;
+  padding-top: $base-padding;
+}
+
+
+.landing-section,
+.landing-section_open {
+  @include respond-to(medium-up) {
+    @include span-columns(5);
+
+    border-bottom: none;
+  }
+
+  border-bottom: $thin-border-size solid $mid-gray;
```

Comment:
Expected item on line 16 to appear before line 10. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets